.eslintrc.cjs

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
eslint uses `require`, even if you set `type: module` in `package.json`, so I had to make this a `.cjs` file to make it able to be required by eslint.